src/components/Nav/LoginForm.jsx

```diff
@@ -61,9 +59,10 @@ export default function LoginForm() {
         }
       )
       .then((res) => {
-        console.log(res);
-        localStorage.setItem("token", res.data.access_token);
-        history.push("/");
+        storeLoginInformation(res.data.access_token, username);
+        // kludgily refresh screen
+        // history.push("/");
+        window.open("/", "_self");
```

Comment:
history.push() was already the limit of my knowledge. window.open is next level.